src/modules/commands/discord/urbandictionary.ts

```diff
@@ -37,7 +40,10 @@ export default new Command<discord.Message>({
       .json()
       .then((data) => {
         if (!isRecord(data)) {
-          return Promise.reject(new Error(`Unexpected query return type`))
+          return Promise.reject(
+            new Error(`Unexp    definitionEmbed.setFooter('Interaction ended').setColor('#1D2439')
+          await embedMessage.edit({ embed: definitionEmbed })ected query return type`),
```

Comment:
What is this?

Reply:
Good question maybe a paste gone wrong that I didn't notice